Prep key enums for binary serialization

safern opened the pull request:
Contributes to: https://github.com/dotnet/corefx/issues/31477

@ViktorHofer any ideas on how to test this?

ViktorHofer commented on src/Common/src/System/Drawing/KnownColor.cs:
```diff
@@ -7,6 +7,7 @@
 namespace System.Drawing
 {
     [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
```
Are we continuing to build for netcoreapp20 here?

safern replied:
Yes we are, https://github.com/dotnet/corefx/pull/31142#issuecomment-405980805

safern commented:
> I would produce the blobs first by adding a dummy test case in BinaryFormatterTests.cs which outputs the serialized base64string (BinaryFormatterHelpers.ToBase64String) and run it for both netfx and netcoreapp. Then add the blobs in BinaryFormatterTestData. That should be all.

Ok, great. Thanks!

safern commented:
I added tests except for KnownColor which was not available in netstandard2.0 and the tests are built against netstandard -- I don't think is worth adding the netcoreapp configuration just for that enum type. 

Also I updated the blobs.

safern commented:
Not really, I can revert the change, build again after rebasing and then run UpdateBlobs again.

ViktorHofer commented:
Be careful with UpdateBlobs. We haven't updated it since @Anipik changed the data structure. I would recommend not using it and manually updating the blobs.

safern commented:
> Be careful with UpdateBlobs. We haven't updated it since @Anipik changed the data structure. I would recommend not using it and manually updating the blobs.
>
> At some point we should fix that helper method.

Ok, I'll revert my change -- I guess then we should change the comment saying "to update blobs use UpdateBlobs test" and state something that it shouldn't be randomly used without a reason for updating them.
